contributed/rho_statistics.ipynb

```diff
@@ -0,0 +1,1091 @@
+    " - Measure the \"rho statistics\" -- correlation functions of the PSF and star shapes that diagnose PSF leakage into the cosmic shear signal\n",
+    " - Measure calibration factors that allow us to compare measured shapes to intrinsic shapes\n",
+    " - Correct the galaxy-galaxy shear CF from the imaging data and compare the corrected version to the EGC CF\n",
+    " - Measure the correlation function of the EGC over the same area to further look for selection bias\n",
+    " \n",
+    "Many of these steps borrow heavily from other DC2 tutorials, as listed in the code comments.\n",
+    "\n",
+    "Note that--because we are loading many catalogs and performing a lot of processing on them that may not be vectorized--this notebook can take longer than you might expect to run, a few minutes per tract per catalog just to load the catalogs.\n",
+    "\n",
+    "We begin by loading up the code we'll need. The desc-stack-dev kernel should have the dependencies you need for this tutorial."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import warnings\n",
+    "import sys\n",
+    "# Allow us to import \"utils.fieldRotator\" below\n",
+    "sys.path.insert(0, '../tutorials')\n",
+    "\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import FoFCatalogMatching\n",
+    "import lsst.afw.geom as afw_geom\n",
+    "import lsst.daf.persistence as dp\n",
+    "import GCRCatalogs\n",
+    "from GCR import GCRQuery\n",
+    "from desc_dc2_dm_data import REPOS\n",
+    "from utils.fieldRotator import FieldRotator\n",
+    "\n",
+    "import matplotlib.pyplot as plt\n",
+    "import scipy.optimize as op\n",
+    "\n",
+    "with warnings.catch_warnings():\n",
+    "    # Stile throws up a bunch of matplotlib warnings that we can just ignore\n",
+    "    warnings.filterwarnings('ignore')\n",
+    "    import stile\n",
+    "\n",
+    "figsize_x, figsize_y = plt.gcf().get_size_inches()\n",
+    "\n",
+    "%matplotlib inline"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Getting star shapes\n",
+    "-------------------\n",
+    "\n",
+    "This next object demonstrates how to compute star shapes from the object catalogs.  You will need the measured _moments_: Ixx, Ixy, and Iyy.  These are measured in a reference frame that is local to the tracts and patches that the DM stack uses for analysis.  To compare to the galaxy shapes (which are in sky coordinates, relative to the (ra, dec) direction), we also need to rotate to the sky coordinate system using the butler.  \n",
+    "\n",
+    "To perform this, we'll set up a ShearRotator class that takes in the moment data and reproduces the quantities we want.  Note that, per the notation sometimes used by lensing people (and, in particular, used by Stile), \"sigma\" is a linear measurement of size, and not a measurement error."
```

Comment:
Can you add the note about using desc-stack instead of desc-stack-dev @msimet?

Reply:
I just changed the kernel to be desc-stack, and the directions as well!